CONTRIBUTING.md

```diff
@@ -20,6 +20,10 @@ If your repo has certain guidelines for contribution, put them here ahead of the
 
 - [Mentoring Initiatives](https://git.k8s.io/community/mentoring) - We have a diverse set of mentorship programs available that are always looking for volunteers!
 
+## Contact Information
+
+- [Slack channel](https://kubernetes.slack.com/?redir=%2Fmessages%2FCJJ529RUY) - A channel that can be used by people who have questions about the project. It offers an alternative to submitting a Github issue;
```

Comment:
Please add this text as well - 
```
[Join Slack](http://slack.k8s.io) to sign up and join the Kubernetes Slack. Please make sure to read our [Slack Guidelines](https://github.com/kubernetes/community/blob/master/communication/slack-guidelines.md) before participating.
```

Sorry, the link I provided in the GH issue is not the proper one, but a redirect.
```suggestion
-  The [kube-state-metrics slack channel](https://kubernetes.slack.com/messages/CJJ529RUY) provides an effective communication platform to reach out to members and other users of the project. It offers an alternative to submitting a GitHub issue for when you have questions and issues.
```

Reply:
done